apps/settings/js/panels/firefox_sync/panel.js

```diff
@@ -0,0 +1,303 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* global ERROR_UNVERIFIED_ACCOUNT */
+/* global LazyLoader */
+
+define(function(require) {
+  'use strict';
+
+  var SettingsPanel = require('modules/settings_panel');
+  var SettingsUtils = require('modules/settings_utils');
```

Comment:
remember to exclude 'modules/settings_utils' in `js/config/require.js`, or the panel wont work after install with `reset-gaia`